### DISCUSSION TITLE
Release vs Preview?

### DISCUSSION THREAD

**Discussion by JuggernOtt81-USI** [dotnet/maui#7431] [Q&A]
Was happy to see that MAUI is now a RELEASE today. 
unless i misread that. 
but it won't install in VS2022, only VS2022-preview.
if it's a release, why can it only be used in a preview environment?

**Comment by hahyes** [comment]
This is a release of MAUI, not new version of Visual Studio. Releasing MAUI doesn't mean they will release new version of VS too. These are different teams.
